storage/signature.go

```diff
@@ -121,15 +154,39 @@ func (s *Signature) Open(ctx context.Context, fp string) (fs.File, error) {
 			return nil, xerrors.Errorf("read signature manifest: %w", err)
 		}
 
-		signed, err := extensionsign.SignAndZipManifest(s.Signer, manifestData)
+		vsix, err := s.Storage.Open(ctx, filepath.Join(filepath.Dir(fp), vsixPath+".vsix"))
+		if err != nil {
+			// If this file is missing, it means the extension was added before
+			// signatures were handled by the marketplace.
```

Comment:
Looks like this opens the `.vsix`, which should always exist (if not installing will also fail).  Or I could be misreading this :sweat_smile:

Reply:
you are right!

Reply:
This comment refers to loading the vsix though, not the manifest, and the vsix must always exist.  Possibly it was supposed to go in the block above where we read the manifest?

Reply:
Oh yeah probably, the error also says manifest but should probably say the vsix :D